build/images/flow-aggregator/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.15 as flow-aggregator-build
+
+LABEL maintainer="Antrea <projectantrea-dev@googlegroups.com>"
+LABEL description="The docker image for flow aggregator"
```

Comment:
these label statements should come after the *last* `FROM` statement if I recall correctly?